ndk-build/src/manifest.rs

```diff
@@ -216,30 +185,30 @@ where
     seq.end()
 }
 
-fn serialize_catergories<S>(
-    categories: &Option<Vec<String>>,
-    serializer: S,
-) -> Result<S::Ok, S::Error>
+fn serialize_catergories<S>(categories: &Vec<String>, serializer: S) -> Result<S::Ok, S::Error>
 where
     S: Serializer,
 {
-    match categories {
-        Some(categories) => {
-            #[derive(Serialize)]
-            struct Category {
-                #[serde(rename = "android:name")]
-                pub name: String,
-            }
-            let mut c = Vec::new();
-            for category in categories {
-                c.push(Category {
-                    name: category.clone(),
-                });
-            }
-            serializer.serialize_some(&c)
-        }
-        None => serializer.serialize_none(),
+    use serde::ser::SerializeSeq;
+
+    #[derive(Serialize)]
+    struct Category {
+        #[serde(rename = "android:name")]
+        pub name: String,
+    }
+
+    let mut c = Vec::new();
+    for category in categories {
+        c.push(Category {
+            name: category.clone(),
+        });
+    }
```

Comment:
The idea here was that you could just `serialize_element` them directly instead of putting it in a temporary Vec.